app/build.gradle

```diff
@@ -517,6 +532,13 @@ android {
 
     aaptOptions {
         noCompress 'ja'
+        // TODO: This statement should be moved under 'chromium' dimension. So far, these options
+        // are affected to every variants when`isChromiumAvailable()` is true.
+        if (isChromiumAvailable()) {
+            noCompress 'dat'
+            noCompress 'bin'
+            noCompress 'pak'
+        }
```

Comment:
note: I cannot find a good solution to fully explicit these options from other variants. Does anybody has idea?

Reply:
`userProperties.backend` is deprecated #491. Sadly, flavor dimension `backend` doesn't work for aaptOptions as well. I tried to `onVariant` inside `androidComponents` block but it is also failed. I guess that there would be better solution after  upgrading `android_gradle_plugin`.

Reply:
OK, Let's add a TODO here if possible. I'm fine with the way it is, since we don't have any non-chromium assets that would be affected.